task/04-arrays-tasks.js

```diff
@@ -102,7 +103,7 @@ function getArrayOfStrings(arr) {
  *    [ false, 0, NaN, '', undefined ]   => [ ]
  */
 function removeFalsyValues(arr) {
-   throw new Error('Not implemented');
+   return arr.filter(i=>i?i:false);
```

Comment:
ternary is redundant here

Please, see https://javascript.info/logical-operators